### PR TITLE
adb: add French translation

### PR DESCRIPTION
**Pull request by Nico385412**
- [X] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

 - **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/common/adb.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# adb
+
+> Android Debug Bridge: Communiquer avec une instance d'émulateur Android ou un appareil Android.
+> Certaines commandes comme `adb shell` ont leur propre documentation
+> Plus d'informations: <https://developer.android.com/studio/command-line/adb>.
+
+- Vérifie si le processus du serveur adb est en fonctionnement et le démarre
```
Comment (navarroaxel) [inline pages.fr/common/adb.md:7]:
```suggestion
- Vérifie si le processus du serveur adb est en fonctionnement et le démarre :
```

File: pages.fr/common/adb.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+`adb start-server`
+
+- Arrête le processus du serveur adb
```
Comment (navarroaxel) [inline pages.fr/common/adb.md:11]:
```suggestion
- Arrête le processus du serveur adb :
```

File: pages.fr/common/adb.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+`adb kill-server`
+
+- Démarre un shell distant sur l'émulateur/l'appareil ciblé
```
Comment (navarroaxel) [inline pages.fr/common/adb.md:15]:
```suggestion
- Démarre un shell distant sur l'émulateur/l'appareil ciblé :
```

File: pages.fr/common/adb.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+`adb shell`
+
+- Pousse une application Android vers l'émulateur/l'appareil
```
Comment (navarroaxel) [inline pages.fr/common/adb.md:19]:
```suggestion
- Pousse une application Android vers l'émulateur/l'appareil :
```

File: pages.fr/common/adb.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+`adb install -r {{chemin/vers/le/fichier.apk}}`
+
+- Copie un fichier/dossier depuis l'appareil ciblé
```
Comment (navarroaxel) [inline pages.fr/common/adb.md:23]:
```suggestion
- Copie un fichier/dossier depuis l'appareil ciblé :
```

File: pages.fr/common/adb.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+`adb pull {{chemin/vers/le/fichier_ou_dossier_de_l'appareil}} {{chemin/vers/le/dossier_de_destination_local}}`
+
+- Copie un fichier/dossier vers l'appareil ciblé
```
Comment (navarroaxel) [inline pages.fr/common/adb.md:27]:
```suggestion
- Copie un fichier/dossier vers l'appareil ciblé :
```

File: pages.fr/common/adb.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+`adb push {{chemin/vers/le/fichier_ou_dossier_local}} {{chemin/vers/le/dossier_de_destination_de_l'appareil}}`
+
+- Récupère une liste des appareils connectés
```
Comment (navarroaxel) [inline pages.fr/common/adb.md:31]:
```suggestion
- Récupère une liste des appareils connectés :
```

File: pages.fr/common/adb.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
+# adb
+
+> Android Debug Bridge: Communiquer avec une instance d'émulateur Android ou un appareil Android.
+> Certaines commandes comme `adb shell` ont leur propre documentation
```
Comment (navarroaxel) [inline pages.fr/common/adb.md:4]:
```suggestion
> Certaines commandes comme `adb shell` ont leur propre documentation.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @Nico385412, welcome and thanks for your submission. I left some comments.

File: pages.fr/common/adb.md (hunk 1 above)
Comment (Nico385412) [reply pages.fr/common/adb.md]: done 👍

File: pages.fr/common/adb.md (hunk 2 above)
Comment (Nico385412) [reply pages.fr/common/adb.md]: done 👍

File: pages.fr/common/adb.md (hunk 4 above)
Comment (Nico385412) [reply pages.fr/common/adb.md]: done 👍

File: pages.fr/common/adb.md (hunk 6 above)
Comment (Nico385412) [reply pages.fr/common/adb.md]: done 👍

File: pages.fr/common/adb.md (hunk 5 above)
Comment (Nico385412) [reply pages.fr/common/adb.md]: done 👍

File: pages.fr/common/adb.md (hunk 3 above)
Comment (Nico385412) [reply pages.fr/common/adb.md]: done 👍

File: pages.fr/common/adb.md (hunk 8 above)
Comment (Nico385412) [reply pages.fr/common/adb.md]: done 👍

File: pages.fr/common/adb.md (hunk 7 above)
Comment (Nico385412) [reply pages.fr/common/adb.md]: done 👍

**Review by navarroaxel — APPROVED** [review]

File: pages.fr/common/adb.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
+# adb
+
+> Android Debug Bridge: Communiquer avec une instance d'émulateur Android ou un appareil Android.
+> Certaines commandes comme `adb shell` ont leur propre documentation.
```
Comment (marchersimon) [inline pages.fr/common/adb.md:4]: @Nico385412 would be great if you could add this phrase to the [translation template](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md#fr).

**Review by marchersimon — APPROVED** [review]
Thanks @Nico385412

File: pages.fr/common/adb.md (hunk 9 above)
Comment (Nico385412) [reply pages.fr/common/adb.md:4]: done :D 
https://github.com/tldr-pages/tldr/pull/6820

**Pull request merged**